Formula/sleepwatcher.rb

```diff
@@ -36,32 +36,21 @@ def install
       mv "../sleepwatcher.8", "."
       system "make", "install", "PREFIX=#{prefix}"
     end
+  end
 
-    # Write the sleep/wakeup scripts
-    (prefix + "etc/sleepwatcher").install Dir["config/rc.*"]
-
-    # Write the launchd scripts
-    inreplace Dir["config/*.plist"], "/usr/local/sbin", HOMEBREW_PREFIX/"sbin"
-
-    inreplace "config/de.bernhard-baehr.sleepwatcher-20compatibility.plist",
-      "/etc", etc/"sleepwatcher"
-
-    prefix.install Dir["config/*.plist"]
+  service do
+    run [HOMEBREW_PREFIX/"sbin/sleepwatcher", "-V", "-s", "#{Dir.home}/.sleep", "-w", "#{Dir.home}/.wakeup"]
```

Comment:
This should work 

```suggestion
    run [sbin/"sleepwatcher", "-V", "-s", "#{Dir.home}/.sleep", "-w", "#{Dir.home}/.wakeup"]
```

Reply:
Does not work:
Error: Failed to install service files
Error: undefined local variable or method `sbin' for #<Homebrew::Service:0x00007fb6dbaea800>
Did you mean?  bin
Please report this issue:
  https://docs.brew.sh/Troubleshooting
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/sleepwatcher.rb:42:in `block in <class:Sleepwatcher>'
/usr/local/Homebrew/Library/Homebrew/service.rb:169:in `instance_eval'
/usr/local/Homebrew/Library/Homebrew/service.rb:169:in `command'
/usr/local/Homebrew/Library/Homebrew/extend/os/mac/caveats.rb:40:in `plist_caveats'
/usr/local/Homebrew/Library/Homebrew/caveats.rb:43:in `caveats'
/usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/lib/ruby/2.6.0/forwardable.rb:224:in `empty?'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:744:in `caveats'
/usr/local/Homebrew/Library/Homebrew/formula_installer.rb:795:in `finish'
/usr/local/Homebrew/Library/Homebrew/install.rb:304:in `install_formula'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:207:in `block in install'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:205:in `each'
/usr/local/Homebrew/Library/Homebrew/cmd/install.rb:205:in `install'
/usr/local/Homebrew/Library/Homebrew/brew.rb:122:in `<main>'